src/themes/light.scss

```diff
@@ -17,6 +22,11 @@
   --wxc-modal-background: #{$wxc-white-alpha-100};
   --wxc-modal-border: 1px solid #{$wxc-black-alpha-11};
   --wxc-modal-box-shadow: 0 1.5rem 3rem #{$wxc-black-alpha-20}, 0 0 0.0625rem #{$wxc-black-alpha-12};
+  
+  --wxc-spinner-circle-color: #{$wxc-light-gray};
```

Comment:
the spinner colors are already present in this file (line 74). Please remove them from here. Same for video-background